calyx-opt/src/passes/top_down_compile_control.rs

```diff
@@ -814,12 +1000,12 @@ impl Schedule<'_, '_> {
 pub struct TopDownCompileControl {
     /// Print out the FSM representation to STDOUT
     dump_fsm: bool,
-    /// Output a JSON FSM representation to file if specified
-    dump_fsm_json: Option<OutputFile>,
```

Comment:
Is there a reason you decided to get rid of the `dump_fsm_json` option?

Reply:
Yeah, I figured we didn't care too much about correctness of the actual FSM, and instead whether the query hardware looked right. Should I add `dump-fsm` back?

Reply:
That was a mistake, sorry about that! I'll fix that asap.